src/js/zarith.js

```diff
@@ -452,7 +462,17 @@ function ml_z_to_bits(z1, z2) {
 }
 
 // external of_bits: string -> t = "ml_z_of_bits"
-// Should provide ml_z_of_bits const
-function ml_z_of_bits(z1, z2) {
-
+//Provides: ml_z_of_bits
+//Requires: bigInt
+function ml_z_of_bits(arg){
+	var zbase = bigInt(256)
+	var acc = bigInt(0);
+	var pow = bigInt(1);
+	for(var i=0; i<arg.l; i++){
+		var base_bump = bigInt(arg.c.charCodeAt(i));
```

Comment:
Are we sure that all characters in `arg.c` are 0 ≤ … < 256?

Reply:
We aren't. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/charCodeAt It was merely a peculiarity of my use-case.

I'll add a check for that. What sort of behaviour would you want to happen when this is not the case?

Reply:
I fixed this in the new commit.
I went for a validation pass first. I'm not sure if that's preferable or not (w.r.t. performance in particular).
I went for raising `Invalid_argument`. Is that ok with you?

Let me know if these choices are ok. If so, I'll make a small test-suite/series of examples.